build/webpack.common.js

```diff
@@ -39,7 +38,7 @@ const withHTMLPlugin = returnPluginWithConfig(HtmlPlugin, {
   template: `${BOOTSTRAP_DIR}/index.html`, // source template
   title: UI_TITLE, // HTML title
   favicon: `${IMAGES_DIR}/favicon.ico`, // favicon for this page
-  inject: 'head', // any/all built content linked into HTML head element
+  inject: true,
```

Comment:
Need standard inject otherwise the script runs before the `body` has rendered. @matthew-chirgwin we still need to discuss this.

Reply:
Lets chat - more curiosity than anything else!

Reply:
Had a chat - as mentioned, was curious this was a requirement for React. Happy with this change, but @nictownsend was going to timebox if there were options to keep scripts in the `head`